### PR TITLE
Fix rake task names

### PR DESCRIPTION
**Pull request by robwise**
For some reason, dynamically defining methods doesn't get evaluated when doing `rake -T`, so we have to define them explicitly.

[Review on Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/229)

### CODE REVIEW

File: spec/dummy/Gemfile.lock [hunk 1]
```diff
@@ -5,7 +5,7 @@ PATH
       connection_pool
       execjs (~> 2.5)
       rails (>= 3.2)
-      rainbow (~> 2.0)
+      rainbow (~> 2.1)
```
Comment (aaronvb) [inline spec/dummy/Gemfile.lock:8]: @robwise Thanks for catching this.

**Comment by aaronvb** [comment]
@robwise :+1:

**Comment by justin808** [comment]
One comment

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[rakelib/example_type.rb, line 91 \[r2\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/229#-K8umaEMm7uCwu4CfiXY:-K8umaEMm7uCwu4CfiXZ:340819188) ([raw file](https://github.com/shakacode/react_on_rails/blob/3010a0f66123cd05c02bb96bf6239d8831a446ff/rakelib/example_type.rb#L91)):*
this comment does not look right -- short?

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/react_on_rails/229#-:-K8ummwkncxThCZ0WZxS:1458970720)*

**Comment by robwise** [comment]
Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[rakelib/example_type.rb, line 91 \[r2\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/229#-K8umaEMm7uCwu4CfiXY:-K8uqz2t4YsSk_INh2wX:-1116821310) ([raw file](https://github.com/shakacode/react_on_rails/blob/3010a0f66123cd05c02bb96bf6239d8831a446ff/rakelib/example_type.rb#L91)):*
woops, nice catch, fixed!

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/react_on_rails/229)*

**Comment by justin808** [comment]
:shipit:

**Comment by justin808** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/react_on_rails/229)*

**Pull request merged**